reactjs-films-homework/src/components/MovieCard/__snapshots__/MovieCard.test.tsx.snap

```diff
@@ -0,0 +1,90 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
you frequently use snapshot, I suppose you can leave one test for example and get rid from other instances. Ip production code I totally recommend DON't use snapshot testing. Use target testing.